### PR TITLE
Quick hack to get the gamestate inspector working again.

### LINKED ISSUE #7565: [BUG] [Gamestate Inspector] Viewing Unit Data in Inspector UI shows Nothing
### Screenshot
image

### Expected Behaviour
I see the unit data code

### Observed Case
This was HttT for me. Might be anything.

### Version Data
1. Wesnoth 1.17.16+dev (latest)
2. Github Download
3. Windows 10 (msys2)

### Extra Info
the vertical scroll is there, indicating the code is there...just not visible.

### PR DESCRIPTION
**Pull request by Pentarctagon**
This doesn't fix the underlying issue with labels trying to show text too long to be rendered, but it does at least make the text show up now.

Fixes #7565, not sure about the other inspector related issues.

### CODE REVIEW

**Comment by Wedge009** [comment]
I only reliably encounter #7565 in Windows. I used the same HttT starting scenario for testing. Reconfirmed the issue before change and no apparent difference after the change. As best as I can tell, aside from renaming the variable, this is halving the hard-coded limit, making it proportional to `pixel_scale()`, and removing the `static const`.

I'm not sure how this helps your case but it seems to make no difference for me...

**Comment by Pentarctagon** [comment]
Well, before when I clicked a unit it wasn't showing any text and I got the error mentioned in the issue, after the change the text shows up when clicking a unit. If you keep decreasing the limit does text eventually show up?

**Comment by CelticMinstrel** [comment]
Wasn't there also a crash sometimes if the text is too long?

**Comment by Wedge009** [comment]
> I only reliably encounter #7565 in Windows.

Okay, so maybe #7565 and #7093 really are different in that the first shows no text at all and the text only disappears in the second after scrolling. So I can only observe #7093, I can't replicate #7565 at all.

> If you keep decreasing the limit does text eventually show up?

I tried length of 2000 and no difference. So probably different issue.

> Wasn't there also a crash sometimes if the text is too long?

Sounds like #7201.

File: src/gui/dialogs/gamestate_inspector.cpp [hunk 1]
```diff
@@ -128,7 +129,7 @@ class gamestate_inspector::model
 			pages.emplace_back(start, data.size() - start);
 		}
 	}
-	static const unsigned int max_inspect_win_len = 20000;
+	unsigned int page_characters = 10000 / preferences::pixel_scale();
```
Comment (stevecotton) [inline src/gui/dialogs/gamestate_inspector.cpp:132]: Needs to be `video::get_pixel_scale()`, as the `preferences` can also be set to automatic scaling.

**Comment by stevecotton** [comment]
Agreed, it's a hack but it's better than the status quo.

**Comment by CelticMinstrel** [comment]
So does this also fix the crash?

**Comment by Pentarctagon** [comment]
I don't know if it fixes the crash - I was never able to reproduce that.

File: src/gui/dialogs/gamestate_inspector.cpp (hunk 1 above)
Comment (Pentarctagon) [reply src/gui/dialogs/gamestate_inspector.cpp]: Updated.

**Comment by stevecotton** [comment]
I think it fixes the crash, explanation is in https://github.com/wesnoth/wesnoth/issues/7201#issuecomment-1343697267 .

**Pull request merged**